Gulpfile.js

```diff
@@ -107,6 +112,7 @@ const projects = [
         cwd: "packages/table/",
         dependencies: ["core"],
         copy: false,
+        isotest: true,
```

Comment:
That's understood, but aren't the *config flags* used to auto-generate the tasks for each package?  I'll have to look it again. The build system is non-trivial